src/common/colors.js

```diff
@@ -14,18 +14,21 @@ export default {
 
 export const COLOR_MAP = {
   ORANGE: {
-    BASE: '#FFAD16',
+    BASE: '#FFC900',
     LIGHT: '#FFC020',
     DARK: '#933500',
   },
+  ORANGE_DARK: {
+    BASE: '#FF9600',
```

Comment:
why doesnt this need light/dark?

Reply:
The designs don't have light or dark versions of this color, do we need them?

Reply:
I don't really know the history of BASE/LIGHT/DARK, but In general, I think we're trying to avoid having multiple shades of a color on the site (I did some unifications of the greens a while ago at Josh's request).